frontend/src/components/homeview/homeview_components/DashboardStudyKing.jsx

```diff
@@ -11,16 +14,24 @@ class DashboardStudyKing extends Component {
       //[20, 23],
     ],
   };
+
+  componentDidMount() {
+    //todo: handle error properly
+    StudyKingService.getStudyKing()
+      .then((data) => this.setState(data))
+      .catch((msg) => console.log(msg));
+  }
+
   render() {
     return (
       <div id="today_study_king_display" className="mt-5 mb-5">
         <h3 style={{ fontSize: "150%" }}>Today's study king:</h3>
         <br />
         <p style={{ fontSize: "120%" }}>
-          <b> 🔥 some_user</b> (with some_time)
+          <b> 🔥 {this.state.winner} </b> (with {this.state.winnerMinutes})
```

Comment:
At the end of line you could add a unit such as "minutes", you might want to convert the winnerMinutes to winnerHours, then it makes more sense to store the winnerMinutes as double instead of string (see my last comment)

Reply:
Great, just same to what I said before .....